External/Plugins/ASCompletion/Model/MemberModel.cs

```diff
@@ -538,4 +538,40 @@ public int Compare(MemberModel a, MemberModel b)
         }
 
     }
+
+    /// <summary>
+    /// Compare members based on import name
+    /// </summary>
+    public class ByImportTypeMemberComparer : IComparer<MemberModel>
+    {
+        public static Int32 CompareImports(String import1, String import2)
+        {
+            // Use case-insensitive comparison.
+            //IComparer cmp = StringComparer.OrdinalIgnoreCase;
+            // Use case-sensitive comparison.
+            IComparer cmp = StringComparer.Ordinal;
```

Comment:
In Haxe, there's also `import ... in` / `import ... as`, is that being taken into account?

```haxe
import foo.bar.ModuleName as Name;
import foo.bar.ModuleName in Name;
```

What about `using`?